src/schema/job-board-application.json

```diff
@@ -140,8 +140,8 @@
             ],
             "additionalProperties": false
         },
-        "TriggerProxyEmailFallback": {
-            "type": "boolean",
+        "TriggerProxyEmail": {
+            "type": "string",
             "description": "",
```

Comment:
is it worth adding in a description of what the email should be? The proxy email that's been created and used for the account creation?